### PR TITLE
Updated demo link in the README

### PR DESCRIPTION
**Pull request by ebbertd**
Changed the README to include a link to a working demo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,14 +30,14 @@ You can find the latest news about the annotation tool on [this page](documentat
 * [Claudio Beffa](beffa.ch)
 
 ### Current Development of this branch
-* [virtUOS, Universit�t Osnabr�ck](http://www.virtuos.uni-osnabrueck.de)
+* [virtUOS, Universität Osnabrück](http://www.virtuos.uni-osnabrueck.de)
```
Comment (JulianKniephoff) [inline README.md]: I'm confused. What exactly is the difference here? Is this some kind of encoding snafu? :thinking:

File: README.md [hunk 2]
```diff
@@ -30,14 +30,14 @@ You can find the latest news about the annotation tool on [this page](documentat
...
 * [ELAN e.V.](http://elan-ev.de)
 
 ### Contact
-* R�diger Rolf, [rrolf@uni-osnabrueck.de](mailto://rrolf@uni-osnabrueck.de), Phone: +49 541 969 6511
+* Rüdiger Rolf, [rrolf@uni-osnabrueck.de](mailto://rrolf@uni-osnabrueck.de), Phone: +49 541 969 6511
```
Comment (JulianKniephoff) [inline README.md]: Same here. :thinking:

File: README.md (hunk 1 above)
Comment (ebbertd) [reply README.md]: I used the GitHub online editor and apparently it changed the encoding from ISO-8859-1 to UTF-8. Which one do you prefer?

File: README.md [hunk 3]
```diff
@@ -30,14 +30,14 @@ You can find the latest news about the annotation tool on [this page](documentat
...
 
 ## [DEMO](http://entwinemedia.github.com/annotations/)
```
Comment (mtneug) [inline README.md]: @ebbertd this Link needs to be changed, too.

File: README.md (hunk 1 above)
Comment (JulianKniephoff) [reply README.md]: Oh god, so what you're saying is basically that I need to check every file's encoding now? x)

UTF-8 is what we want, I'd say.

**Pull request merged**